### PR TITLE
Set/Unset Before/After steps

### PR DESCRIPTION
**Pull request by duderman**
## Description

Ability to modify before/after steps through the CLI

## Checklist

-   [✅] I have added tests that prove my fix is effective or that my feature works
-   [✅] I have update README (If needed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,42 +97,41 @@ brew install gopinath-langote/one-build/one-build
...
-    project: Containerized Project
-    after: ./clean_database.sh
-    commands:
-      - build: npm run build
+3. Clean up database after some commands
+     ```console
+    1build set after "./clean_database.sh"
     ```
 
+### Removing `before` and `after` commands
```
Comment (gopinath-langote) [inline README.md]: Let's not make a separate section for ```Removing `before` and `after` commands```

The reason behind creating a section for before and after was - to convinence the idea behind it.

Let's have one section of Before and after – and just examples for setting and unsetting it?

WDYT?

**Comment by gopinath-langote** [comment]
Please also check `Codacy` - bot reviews

File: README.md (hunk 1 above)
Comment (duderman) [reply README.md]: sure. sounds reasonable to me. gonna change it now

**Comment by duderman** [comment]
@gopinath-langote where can I see codacy review?

**Comment by gopinath-langote** [comment]
https://app.codacy.com/manual/gopinath-langote/1build/pullRequest?prid=4307405

**Comment by duderman** [comment]
@gopinath-langote should be fixed now ;)

**Comment by gopinath-langote** [comment]
@duderman LGTM.

**Pull request merged**